packages/react-ui-kit/src/Identity/colors.js

```diff
@@ -106,18 +106,3 @@ export const COLOR = {
   ...LIGHT_COLOR,
   ...OPAQUE_COLOR,
 };
-
-export const FONT = {
-  fontFamily: '-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Helvetica, Arial, sans-serif',
-};
```

Comment:
I would argue that we could keep that even if it's used only in one place.. just to be able to find it most likely inside the variables/settings section..

Reply:
nope, we don't need it.

Reply:
:(